### PR TITLE
Introduce support for devices connected over local sockets

### PR DESCRIPTION
**Pull request by anszom**
This PR introduces support for locally-connected devices in moolticuted, aimed to support the "emulator" build of mooltipass mini BLE. 
The code doesn't support other device models at the moment.

### CODE REVIEW

File: src/MPDevice.cpp [hunk 1]
```diff
@@ -49,7 +49,8 @@ MPDevice::MPDevice(QObject *parent):
 
             if (isBLE())
             {
-                bleImpl->readUserSettings(pMesProt->getPayloadBytes(data, 2, 4));
+                if(pMesProt->getMessageSize(data) >= 4) // these settings are sent only when user is logged in
+                    bleImpl->readUserSettings(pMesProt->getPayloadBytes(data, 2, 4));
```
Comment (limpkin) [inline src/MPDevice.cpp]: Awesome catch...

moreover, they are now included in the get_status message... so you we may just remove that.... what do you think @deXol ?

Comment (deXol) [reply src/MPDevice.cpp]: this is the MPCmd::MOOLTIPASS_STATUS command's callback. In line 53 it is reading the user settings from get_status.

Comment (limpkin) [reply src/MPDevice.cpp]: oops, my bad!

**Comment by limpkin** [comment]
merging #642 instead... thanks for the awesome work :)

**Pull request closed without merging**